### PR TITLE
Home Page navigation fixed issue #316

### PR DESCRIPTION
**Pull request by Lakshyyaa**
fixed issue #316

### CODE REVIEW

**Review by narayan954 — CHANGES_REQUESTED** [review]
need to fix that

File: src/App.jsx [hunk 1]
```diff
@@ -197,8 +197,16 @@ function App() {
           alt="dummygram"
           className="app__header__img w-100"
           onClick={() => {
+            if(location.pathname==='/dummygram/login'){
+              navigate("/dummygram/login"); 
+            }
+            else if (location.pathname === '/dummygram/signup'){
+              navigate("/dummygram/signup");               
+            }
+            else{
```
[the remark this answers is not in the document]
Comment (Lakshyyaa) [reply src/App.jsx:206]: yes, here if user's on login/signup and clicks on the logo he/she will be redirected to the same page.
But, if you wish for no navigation action to be performed when user is on login/signup. I have the code ready 
waiting for your merge approval

[the remark this answers is not in the document]
Comment (Lakshyyaa) [reply src/App.jsx]: hey i made a new commit solve it

[the remark this answers is not in the document]
Comment (narayan954) [reply src/App.jsx]: great, mergin

**Review by narayan954 — APPROVED** [review]
looks good to me for now, but actually this implementation is incomplete and hard-coded. Need to fix it with a better approach later

**Pull request merged**